cli/image_list.go

```diff
@@ -4,11 +4,12 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/alibaba/pouch/apis/filters"
 	"github.com/alibaba/pouch/apis/types"
 	"github.com/alibaba/pouch/pkg/reference"
 	"github.com/alibaba/pouch/pkg/utils"
 
-	digest "github.com/opencontainers/go-digest"
```

Comment:
Is this related to goimport version ? @fuweid @HusterWan

Reply:
so far it is ok.